passport/src/auth-handler.ts

```diff
@@ -102,8 +102,7 @@ function createOnCallback(domain: string): express.Handler {
   return (req, res, next) => {
     passport.authenticate(`auth0-${domain}`, (err, user, _info, _extra) => {
       if (err) { return next(err); }
-      // TODO: redirecting to /login, a misconfiguration causes a confusing redirect loop
-      if (!user) { return res.redirect('/login'); }
+      if (!user) { return res.redirect('/login-error'); }
```

Comment:
How does this change the behaviour of existing apps, specifically of current templates with auth and any apps built on top of those?

Reply:
I will submit PRs to update the package in all first-party templates.